tracer/test/Datadog.Trace.Tools.Runner.IntegrationTests/Checks/IisCheckTests.cs

```diff
@@ -0,0 +1,145 @@
+// <copyright file="IisCheckTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_1
+
+using System.Diagnostics;
+using System.IO;
+using System.Net.Http;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using Datadog.Trace.TestHelpers;
+using Datadog.Trace.Tools.Runner.Checks;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.Tools.Runner.IntegrationTests.Checks
+{
+    [Collection(nameof(ConsoleTestsCollection))]
+    public class IisCheckTests : TestHelper, IClassFixture<IisFixture>
+    {
+        private readonly IisFixture _iisFixture;
+
+        public IisCheckTests(IisFixture iisFixture, ITestOutputHelper output)
+            : base("AspNetCoreMvc31", output)
```

Comment:
We run these integration tests separately from the other integration tests right? Otherwise you'll probably want to make sure they don't run in parallel.

Also, just be aware this takes _minutes_ to start up (still need to look into why 😞 )

Reply:
I'm pretty sure they're running sequentially.